utils/internal/download_gigaspeech_from_speechocean.sh

```diff
@@ -0,0 +1,196 @@
+#!/usr/bin/env bash
+# Copyright 2021  Jiayu Du
+#                 Seasalt AI, Inc (Author: Guoguo Chen)
+
+set -e
+set -o pipefail
+
+stage=0
+with_dict=false
+
+. ./utils/parse_options.sh || exit 1
+
+if [ $# -ne 2 ]; then
+  echo "Usage: $0 <gigaspeech-dataset-dir> <host>"
```

Comment:
You already know from the script that you are downloading from speechocean, why do you need the second parameters?